NEWS.md

```diff
@@ -24,6 +24,9 @@ This section lists changes that do not have deprecation warnings.
   * Keyword arguments are processed left-to-right: if the same keyword is specified more than
     once, the rightmost occurrence takes precedence ([#17785]).
 
+  * Broadcasting now also handles tuples and scalars (that is anything other than arrays and tuples),
```

Comment:
```
 * `broadcast` now handles tuples, and treats any argument that is not a tuple or an array as a "scalar" ([#16986]).
```
You also need to run `julia doc/NEWS-update.jl` to add a link for the issue number.